README.md

```diff
@@ -1,81 +1,2 @@
-- https://www.nerdstickers.com.br/todos/node-js/
-- https://www.nerdstickers.com.br/todos/javascript-ii/
-- https://www.nerdstickers.com.br/todos/html5/
-- https://www.nerdstickers.com.br/todos/css3/
-- https://www.nerdstickers.com.br/todos/bootstrap-i/
-- https://www.nerdstickers.com.br/todos/octocat-ii/
-- https://www.nerdstickers.com.br/todos/vscode/
-- https://www.nerdstickers.com.br/todos/computer-engineer/
-- https://www.nerdstickers.com.br/todos/hp-world-ii/
-- https://www.nerdstickers.com.br/todos/hp-world-vi/
-- https://www.nerdstickers.com.br/todos/adesivo-i-fucking-love/
-- https://www.nerdstickers.com.br/todos/adesivo-heart-waves/
-- https://www.nerdstickers.com.br/todos/mario-bros/
-- https://www.nerdstickers.com.br/todos/golang/
-- https://www.nerdstickers.com.br/todos/raspberry-pi/
-- https://www.nerdstickers.com.br/todos/docker-i/
-- https://www.nerdstickers.com.br/todos/bitbucket-novo/
-- https://www.nerdstickers.com.br/todos/kubernetes/
-- https://www.nerdstickers.com.br/todos/debian-i/
-- https://www.nerdstickers.com.br/todos/google-cloud-plataform/
-- https://www.nerdstickers.com.br/todos/code-and-beers/
-- https://www.nerdstickers.com.br/todos/c/
-- https://www.nerdstickers.com.br/todos/docker-i/
-- https://www.nerdstickers.com.br/todos/golang/
-- https://www.nerdstickers.com.br/todos/node-js-ii/
-- https://www.nerdstickers.com.br/todos/javascript/
-- https://www.nerdstickers.com.br/todos/mongodb/
-- https://www.nerdstickers.com.br/todos/rails/
-- https://www.nerdstickers.com.br/todos/flutter/
-- https://www.nerdstickers.com.br/todos/vscode/
-
-# [@camilaperes](https://github.com/camilaperes)
-- https://www.nerdstickers.com.br/todos/m-maravilha-ii/
-- https://www.nerdstickers.com.br/todos/tux-linux/
-- https://www.nerdstickers.com.br/todos/linux-mint/
-- https://www.nerdstickers.com.br/todos/poke-pixel-i/
-- https://www.nerdstickers.com.br/todos/poke-pixel-iii/ 
-
-# [@guiwb](https://github.com/guiwb)
-- https://www.nerdstickers.com.br/todos/heisenberg-ii/
-- https://www.nerdstickers.com.br/todos/cs-go/
-- https://www.nerdstickers.com.br/todos/laravel-ii/
-- https://www.nerdstickers.com.br/todos/vscode/
-- https://www.nerdstickers.com.br/todos/css-is-awesome/
-- https://www.nerdstickers.com.br/todos/octocat/
-- https://www.nerdstickers.com.br/todos/git/
-- https://www.nerdstickers.com.br/todos/there-place-like/
-- https://www.nerdstickers.com.br/todos/heroku/
-- https://www.nerdstickers.com.br/todos/php/
-- https://www.nerdstickers.com.br/todos/slack/
-- https://www.nerdstickers.com.br/todos/javascript/
-
-# [@thefuga](https://github.com/thefuga)
-- https://www.nerdstickers.com.br/todos/c/
-- https://www.nerdstickers.com.br/todos/python/
-- https://www.nerdstickers.com.br/todos/linux-freedom/
-- https://www.nerdstickers.com.br/todos/docker-i/
-- https://www.nerdstickers.com.br/todos/ruby/
+# [@jschneiders](https://github.com/jschneiders)
+- https://www.nerdstickers.com.br/todos/cats-tea/
```

Comment:
Not sure if this belongs here. Maybe you could create a new package dedicated for it?